README.md

```diff
@@ -1,34 +1,42 @@
-[ci-img]: https://img.shields.io/travis/ciena-frost/ember-frost-theme.svg "Travis CI Build Status"
+[ci-img]: https://travis-ci.org/ciena-frost/ember-frost-theme.svg "Build Status"
 [ci-url]: https://travis-ci.org/ciena-frost/ember-frost-theme
-[cov-img]: https://img.shields.io/coveralls/ciena-frost/ember-frost-theme.svg "Coveralls Code Coverage"
+
+[cov-img]: https://coveralls.io/repos/github/ciena-frost/ember-frost-theme/badge.svg?branch=master "Code Coverage"
 [cov-url]: https://coveralls.io/github/ciena-frost/ember-frost-theme
+
 [npm-img]: https://img.shields.io/npm/v/ember-frost-theme.svg "NPM Version"
 [npm-url]: https://www.npmjs.com/package/ember-frost-theme
 
-# ember-frost-theme
+# ember-frost-theme <br /> [![Travis][ci-img]][ci-url] [![Coveralls][cov-img]][cov-url] [![NPM][npm-img]][npm-url]
 SASS based default theme variables for Frost projects
 
-# usage
-
-`ember install ember-frost-theme`
+# Usage
 
-Modify your ember-cli-build.js to include 
 
 ```
-var frostTheme = require('frostTheme');
+ember install ember-frost-theme
+```
+
 
-var app = new EmberAddon(defaults, {
+Modify your `ember-cli-build.js` to include:
+
+
+```javascript
+const frostTheme = require('frostTheme')
+
+const app = new EmberAddon(defaults, {
     sassOptions: {
       includePaths: [
         frostTheme.includePaths
```

Comment:
Can you update it with the *correct* way, since I haven't consumed it yet? (in another PR)